lib/apaka/packaging/templates/etc-pbuilderrc

```diff
@@ -78,7 +78,11 @@ fi
 # Append additional packages here
 # cmake required by quite some package, including qtbindings
 # apt-utils, e.g., for running apt-ftparchive in the hook dir see https://wiki.debian.org/PbuilderTricks
-EXTRA_PACKAGES+="apt,cmake,dh-autoreconf,libgl1-mesa-dev,libboost-dev,yard,gem2deb"
+
+# perl-openssl-defaults is needed due to the bug in the debootstrap: https://bugs.launchpad.net/ubuntu/+source/debootstrap/+bug/86536?comments=all
+# the issue arrised by building qtbindings
+
+EXTRA_PACKAGES+="apt,cmake,dh-autoreconf,libgl1-mesa-dev,libboost-dev,yard,gem2deb,perl-openssl-defaults"
```

Comment:
Note that this will affect all generated packages. Is this really needed for all or just a particular package: The bug report is referring to all packages that require a virtual perl-openssl-abi-1.1?

Reply:
I agree, perl-openssl-abi-1.1 will be installed by every packaging process. I thought, since other packages are not related to it. It will have no affect on it. Or do I miss smth?

According to the bug forum, this issue happens to any package, that depends on a virtual package. So I assume, that any dep that adds virtual perl-openssl-abi-1.1, will fail. 
But in our case, it is only qtbinding.

Is there other way to add dep just in case if qtbindings should be built?

Reply:
Sorry, I was mistaken. Just means, the base image will contain the package by default, which will be ok.